# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/openoffice.org-3.1.0.oooXXXXX.gcc44.buildfixes.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- OOO310_m1.orig/sw/source/core/doc/docbm.cxx	2009-02-13 09:42:48.000000000 +0000
+++ OOO310_m1/sw/source/core/doc/docbm.cxx	2009-02-13 09:42:56.000000000 +0000
@@ -334,7 +334,7 @@
     return pA==pB;    
 }
 */
-
+#if 0
 // TODO not finished yet, still neet to add this check
 bool _checkFieldBookmarkSanity(const SwDoc *pDoc) 
 {
@@ -355,7 +355,7 @@
     }
     return true;
 }
-
+#endif
 void SwDoc::deleteBookmark( /*[in]*/sal_uInt16 nPos )
 {
 	SwBookmark *pBM = (*pBookmarkTbl)[nPos];
--- OOO310_m1.orig/sw/source/filter/ww8/WW8TableInfo.cxx	2009-02-13 10:25:13.000000000 +0000
+++ OOO310_m1/sw/source/filter/ww8/WW8TableInfo.cxx	2009-02-13 10:25:21.000000000 +0000
@@ -32,6 +32,7 @@
 #include "precompiled_sw.hxx"
 
 #include <iostream> 
+#include <stdio.h> 
 #include "WW8TableInfo.hxx"
 #include "swtable.hxx"
 #include "pam.hxx"
--- OOO310_m1.orig/sc/source/filter/ftools/ftools.cxx	2009-02-13 11:14:25.000000000 +0000
+++ OOO310_m1/sc/source/filter/ftools/ftools.cxx	2009-02-13 11:14:37.000000000 +0000
@@ -39,6 +39,7 @@
 #include <sot/storage.hxx>
 
 #include <math.h>
+#include <stdio.h>
 #include "global.hxx"
 #include "document.hxx"
 #include "stlpool.hxx"
Index: mkdepend/def.h
===================================================================
--- openoffice.org.orig/soltools/mkdepend/def.h	(revision 269360)
+++ openoffice.org/soltools/mkdepend/def.h	(working copy)
@@ -154,7 +154,7 @@
 
 char			*copy();
 char			*base_name();
-char			*getline();
+char			*get_line();
 char			*isdefined();
 struct filepointer	*getfile();
 struct inclist		*newinclude();
Index: mkdepend/main.c
===================================================================
--- openoffice.org/soltools/mkdepend/main.c	(revision 269360)
+++ openoffice.org.orig/soltools/mkdepend/main.c	(working copy)
@@ -548,7 +548,7 @@
  * Get the next line.  We only return lines beginning with '#' since that
  * is all this program is ever interested in.
  */
-char *getline(filep)
+char *get_line(filep)
 	register struct filepointer	*filep;
 {
 	register char	*p,	/* walking pointer */
Index: mkdepend/parse.c
===================================================================
--- openoffice.org.orig/soltools/mkdepend/parse.c	(revision 269360)
+++ openoffice.org/soltools/mkdepend/parse.c	(working copy)
@@ -45,7 +45,7 @@
 	register int	type;
 	boolean recfailOK;
 
-	while ((line = getline(filep))) {
+	while ((line = get_line(filep))) {
 		switch(type = deftype(line, filep, file_red, file, TRUE, symbols)) {
 		case IF:
 		doif:
@@ -168,7 +168,7 @@
 	register char	*line;
 	register int	type;
 
-	while ((line = getline(filep))) {
+	while ((line = get_line(filep))) {
 		switch(type = deftype(line, filep, file_red, file, FALSE, symbols)) {
 		case IF:
 		case IFFALSE:
